Dockerfile-ut-func

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:20.04 AS base
 RUN apt-get update \
   && apt-get install -y curl make g++ \
   && apt-get install -y cpio \
-  && curl -sL https://deb.nodesource.com/setup_14.x | bash \
+  && curl -sL https://deb.nodesource.com/setup_18.x | bash \
```

Comment:
Lets not touch Dockerfile-ut-func (this for user transform we should have discussion with Services team before doing this)

Reply:
We've taken them into confidence for this project. They'll have to complete their side of testing before green-lighting the release.

Reply:
CC @Jayachand @krishna2020

Reply:
Ok make sense